default-fields/0000-default-fields.md

```diff
@@ -6,31 +6,206 @@
 # Summary
 [summary]: #summary
 
-One para explanation of the feature.
+Allow struct definitions to supply default values for individual fields,
+and then allow those fields to be omitted from struct initialisation:
+
+```rust
+struct Foo {
+    a: &'static str,
+    b: bool = true,
+    c: i32,
+    d: Vec<i32> = Vec::default()
```

Comment:
`default()` is not in the prelude so we would need `default()` to be syntatic sugar in struct declarations. That would be assymetrical to normal uses of `default()`, type inference on lhs would be symetrical to how `let` works.